extensions/BugModal/web/bug_modal.js

```diff
@@ -1419,43 +1419,46 @@ $(function() {
             saveBugComment(event.target.value);
         });
 
-    // Allow to attach pasted text directly
-    document.querySelector('#comment').addEventListener('paste', async event => {
-      const text = event.clipboardData.getData('text');
-      const lines = text.split(/(?:\r\n|\r|\n)/).length;
-
-      if (lines < 50) {
-        return;
-      }
-
-      const extract = text.replace(/(?:\r\n|\r|\n)/, ' ').trim().substr(0, 20);
-      const summary = (window.prompt(
-        `You’re pasting ${lines} lines of text starting with “${extract}”. ` +
-        'Enter the summary below and click OK to upload it as an attachment. Click Cancel to paste it normally.'
-      ) || '').trim();
-
-      if (!summary) {
-        return;
-      }
-
-      event.preventDefault();
+    const comment = document.querySelector('#comment');
+    if (comment.classList.contains('allow-attach-paste')) {
```

Comment:
I think this is maybe backwards. I think we should look for the class *not* present when presenting the confirmation dialog.

```js
if (!comment.classList.contains('allow-attach-paste')) {
```

This way if we never want to see the confirmation dialog and just always allow pasting of any size text, the user preference 'ui_allow_attach_paste' will be on. So if the js sees the class, then it skips the whole code block.